Cargo.toml

```diff
@@ -4,8 +4,11 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-lambda_http = "0.8.3"
+axum = "0.7.2"
+#lambda_http = "0.8.3"
+lambda_http = { git = "https://github.com/awslabs/aws-lambda-rust-runtime", branch = "hyper1_upgrade" }
```

Comment:
Temporarily using an unreleased version due to an ongoing support: https://github.com/awslabs/aws-lambda-rust-runtime/issues/738